pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/NotificationUtil.java

```diff
@@ -202,12 +205,12 @@ public void sentAbstractionExpertQuestion(@NonNull String from, @NonNull String
         Recipient emailRecipient = new Recipient(to);
         emailRecipient.setPersonalization(mapy);
         try {
-            SendGridClient sendGridClient = new SendGridClient();
+            SendGridClient sendGridClient = new SendGridClient(SendGridFactory.createSendGridInstance(emailKey, emailProxyUrl));
             JsonObject emailClientSettings = new JsonObject();
             emailClientSettings.addProperty("sendGridFrom", from);
             emailClientSettings.addProperty("sendGridFromName", name);
-            sendGridClient.configure(emailKey, new Gson().fromJson(emailClientSettings.toString(), JsonObject.class));
-            sendGridClient.sendSingleEmail(sendGridTemplate, emailRecipient);
+            sendGridClient.sendSingleEmail(sendGridTemplate, emailRecipient,
+                    new Gson().fromJson(emailClientSettings.toString(), JsonObject.class));
         } catch (Exception ex) {
             logger.error("An error occurred trying to send abstraction expert question.", ex);
```

Comment:
Is this error message always correct? Is there a reason we are logging an error rather than throwing? (I don't know the calling context.)

Reply:
I think this error message is only correct for one of the usecases of the emails, not all. Most probably we forgot to update this while new usecases were being added to Notification class

Reply:
I'm slightly terrified of changing this from logging to a throw since this gets called from a lot of patch-related endpoints.